integration-tests/tests/run-analysis-azure.ts

```diff
@@ -97,14 +97,11 @@ const testRunAnalysisAzure = _.flowRight(
 
   // Save notebook to avoid "unsaved changes" modal when test tear-down tries to close the window
   await click(frame, '//button[starts-with(@title, "Save and create checkpoint")]');
-
-  // Cleanup: rely on workspace deletion to delete the runtime. "Deleting" status is not considered deletable by Leonardo,
-  // so deletes are not idempotent (a second request to delete will trigger an error 409 and block workspace deletion)
 });
 
 registerTest({
   name: 'run-analysis-azure',
   fn: testRunAnalysisAzure,
-  targetEnvironments: ['dev'],
+  targetEnvironments: ['dev', 'staging'],
```

Comment:
I will disable this on staging once I have this test isolated and running on its own Github action which can be separately scheduled.